src/uwtools/drivers/forecast.py

```diff
@@ -185,3 +166,23 @@ def stage_files(
                 shutil.copyfile(src_path, dst_path)
             msg = f"File {src_path} staged in run directory at {dst_fn}"
             logging.info(msg)
+
+    # Private methods
+
+    def _create_model_config(self, base_file: str, outconfig_file: str) -> None:
```

Comment:
It may turn out that this method should be public, but starting private and upgrading to public is the conservative choice.